src/Horarium.AspNetCore/RegistrationHorariumExtension.cs

```diff
@@ -60,9 +60,9 @@ public static void StartHorariumServer(this IServiceProvider serviceProvider)
 
         private static void PrepareSettings(HorariumSettings settings, IServiceProvider serviceProvider)
         {
-            if (settings.JobFactory is DefaultJobFactory)
+            if (settings.JobExecutionContextFactory is DefaultJobExecutionContextFactory)
```

Comment:
Name of settings is very difficult :(

Reply:
What about `IJobScopeFactory` (and `IJobScope`)?

Reply:
IJobScopeFactory is better 👍